CONTRIBUTING.md

```diff
@@ -56,6 +56,39 @@ So this approach has been abandoned a while ago.
 
 ### Releasing LALRPOP
 
+#### Semver policy
+
+All releases should follow rust's semantic versioning guidelines, as described
+at https://doc.rust-lang.org/cargo/reference/semver.html. Breaking changes in
+our context include changes that break previously working lalrpop grammars.
+
+If a PR introduces a breaking change, the breaking-ness should not delay the
+merge of the PR.  Instead, master will become breaking and the next release
+from master will be a breaking change release.  However, to support continued
+non-breaking releases and reduce the frequency of breaking releases, we will
+create a branch off of the last commit before the breaking change and backport
+future non-breaking changes to it.  For example, if the latest release was 0.22
+and a breaking change is merged, we will create a branch named "0.22.x" from
+the commit prior to the breaking change.  We will backport to that branch as
+non-breaking changes are merged, and possible release 0.22.1, 0.22.2 etc from
+it.
+
+Except in rare and severe circumstances, the 0.22.x branch will only be
+maintained until the release of version 0.23.  A critical bug may warrant a
+later backport to the 0.22.x series, but in general, we will stop backporting
+and focus efforts on the 0.23 series as soon as 0.23 is released.
```

Comment:
Just to clarify there is only 1 0.22.x branch being maintained and not a 0.22.1, 0.22.2, etc